src/InfluxDB.LineProtocol/LineProtocolWriter.cs

```diff
@@ -41,7 +41,21 @@ public LineProtocolWriter Measurement(string name)
                     throw InvalidPositionException($"Cannot write new measurement \"{name}\" as no field written for current line.");
             }
 
-            textWriter.Write(EscapeName(name));
+            foreach (char c in name)
```

Comment:
Measurement names don't need to escape `'='` only commas and spaces. It was also slightly faster to do not check for `'='` so I though it worth it for correctness and speed.